### PR TITLE
theme-color: documenting requirements detail

### LINKED ISSUE #25: audit: Omnibox matches brand colors
Manual evaluation:  manifest has  `theme_color`, `<head>` has `<meta name="theme-color" content="_any valid CSS color_">`

##### input:
* parsed manifest - #22

##### audit:
* parse manifest for `theme_color`. 
* Grab DOM and parse `document.head` for the `theme-color` meta.

Currently both are required. Chrome will be transitioning to use manifest theme-color by default: go/fizz-feature-backlog has a tracking item for this.

See also "PWA validator" bookmarklet - #17

_Docs_:  [Support for theme-color in Chrome 39 for Android](https://developers.google.com/web/updates/2014/11/Support-for-theme-color-in-Chrome-39-for-Android?hl=en), and [whatwg/meta-theme-color: Spec for the `theme-color` meta extension](https://github.com/whatwg/meta-theme-color)

### PR DESCRIPTION
**Pull request by paulirish**
just a small docs addition. fixes #25

### CODE REVIEW

File: aggregators/omnibox-is-themed/index.js [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 const Aggregate = require('../aggregate');
 
-class AddToHomescreen extends Aggregate {
+class OmniboxThemeColor extends Aggregate {
```
Comment (samccone) [inline aggregators/omnibox-is-themed/index.js]: https://github.com/GoogleChrome/lighthouse/blob/1e1df34b8343171bb628983a93100c9fd45360f9/aggregators/will-get-add-to-homescreen-prompt/index.js

^ will need to update here?

Comment (brendankenny) [inline aggregators/omnibox-is-themed/index.js]: I think this is fixing a mistake from when Paul split up the old `AddToHomescreen`.

On master, both `aggregators/omnibox-is-themed/index.js` and `aggregators/will-get-add-to-homescreen-prompt/index.js` contain a class named `AddToHomescreen`

Comment (samccone) [inline aggregators/omnibox-is-themed/index.js]: kk I was very confused :)

Comment (paulirish) [inline aggregators/omnibox-is-themed/index.js]: Ya what he said

**Pull request merged**